localization/english/culture/wc_base_innovations_l_english.yml

```diff
@@ -0,0 +1,68 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ #####################
+ # High Medieval Era #
+ #####################
+ 
+ #####################
+ # Late Medieval Era #
+ #####################
+ 
+ #####################
+ # Early Modern Era  #
+ #####################
+ 
+ #Mil
+ innovation_conscripts:0 "Conscripts"
+ innovation_conscripts_desc:0 "Conscription laws allow us to more field a larger and cheaper army."
+ 
+ innovation_better_seige:0 "High Velocity Bombs"
+ innovation_better_seige_desc:0 "Bombs that fly at higher speeds can deal more damage on impact. With the high velocity bombs we can render castles less useful."
+ 
+ innovation_defense_charters:0 "Defense Charters"
+ innovation_defense_charters_desc:0 "New defense charters within population centers have allowed our cities to be more well garrisoned and defended."
+ 
+ innovation_guns:0 "Riflemen"
+ innovation_guns_desc:0 "The new era of technology has also brought us a new era of warfare. Though mildly innacurate at larger distances, rifles can pack a massive punch to anyone on the other side of the barrel."
+ 
+ innovation_professional_army:0 "Professional Army"
+ innovation_professional_army_desc:0 "Our army has learned a lot over the decades and we have been able to field a highly trained and experienced army. With such talent among our ranks, there are always those that stand out."
+
+ innovation_recruitment_centers:0 "Recruitment Centers"
+ innovation_recruitment_centers_desc:0 "Recruitment centers allow us to recruit a lot more soldiers into our army. With our new recruitment strategies we are able to reach the most remote areas of our domain to find anyone willing to fight."
+
+ #Civic
+ innovation_black_powder:0 "Black Powder"
+ innovation_black_powder_desc:0 "While the black powder can be used to kill and destroy, it can also be used to help us with the cost and speed of construction."
```

Comment:
```suggestion
 innovation_black_powder_desc:0 "While the black powder can be used to kill and destroy, it can also be used to help us with both the cost and speed of construction."
```